common/beerocks/hostapd/CMakeLists.txt

```diff
@@ -0,0 +1,135 @@
+project(prplmseh_hostapd VERSION ${prplmesh_VERSION})
+
+# Set the base path for the current module
+set(MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
+
+# Support more permissive c++ language
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
```

Comment:
I don't think you're going to use GNU extensions, so don't do this.

Reply:
Removed